qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -17,6 +17,13 @@ def setUpClass(cls):
         cls._rbd_cmd(['create', '--size=1G', 'img1'])
         cls._rbd_cmd(['create', '--size=2G', 'img2'])
 
+        osd_metadata = cls.ceph_cluster.mon_manager.get_osd_metadata()
+        cls.bluestore_support = True
+        for osd in osd_metadata:
+            if osd['osd_objectstore'] != 'bluestore':
+                cls.bluestore_support = False
+                break
```

Comment:
Interestingly, this is very similar to https://github.com/ceph/ceph/pull/20865/files#diff-dbb3a62ac313f00ac7caf30816c208bdR91